dist/js/script.js

```diff
@@ -3,3 +3,82 @@
  * Date: 12 - 06 -2021
  * Copyright (c) 2021 Calvince
  */
+
+(function () {
+  // Utility functions
+
+  // /**
+  //  * @param {string} idValue - an element ID
+  //  * @returns {Object} DOM object associated with id
+  //  * */
+  // function id(idName) {
+  //   return document.getElementById(idName);
+  // }
+
+  /**
+   * @param {string} className - a classs name of the html elements
+   * @returns {Object} Dom object(s) of the associated html elements
+   */
+  function classItem(className) {
+    return document.querySelector(className);
+  }
+
+  /**
+   * @param {string} className - a classs name of the html elements
+   * @returns {Object} Dom objects of the associated html elements
+   */
+  function classItems(className) {
+    return document.querySelectorAll(className);
+  }
+
+  // Variables
+  let isOpen = false;
+  const menu = classItem('.mobile-menu-button');
+  const menuItem = classItem('.mobile-menu-button-item');
+  const nav = classItem('.desktop-menu');
```

Comment:
- [ ] As requested by the previous code reviewer, make sure the menu and other parts of the page matches the original design. 
_See images below._

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/123251830-7ce26600-d4a0-11eb-96fa-9f6b6519d37d.png) | ![image](https://user-images.githubusercontent.com/49621279/123252029-aac7aa80-d4a0-11eb-8709-b5aaf10425f1.png)
![image](https://user-images.githubusercontent.com/49621279/123252120-cc289680-d4a0-11eb-9a83-a058c5c13744.png) | ![image](https://user-images.githubusercontent.com/49621279/123252185-dd71a300-d4a0-11eb-9506-43ff4bf47b0d.png)